apps/judicial-system/web/src/routes/Prosecutor/RestrictionRequest/Overview/Overview.tsx

```diff
@@ -92,7 +99,7 @@ export const Overview: React.FC = () => {
 
       setModalVisible(true)
     } catch (e) {
-      // TODO: Handle error
+      toast.error(formatMessage(errors.general))
```

Comment:
Consider removing try catch here as each remote call has its own error handling in the useCase hook.